app/renderer/components/bottombar.js

```diff
@@ -0,0 +1,27 @@
+"use strict";
+
+const AppStatus = require('../app_status');
+
+module.exports = {
+    
+    props: ['suite'],    
```

Comment:
Is the suite prop used in the component.

In vue.js props are the standard way of passing data from parent component to child component, you should only pass data that you actually need for rendering